### PR TITLE
Remove app / component to add sample data

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description

- Removed all the buttons that redirect to the OpenSearch sample data.
- Removed the routes to this app / component.
- Accommodated tests & test snapshots.
 
### Issues Resolved
- #19 

<details>

<summary>Screenshots</summary>

image
image
image
image

</details>

 
### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
  - [x] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by yenienserrano** [comment]
`Build and test / Run functional tests (6) (push)`  is failing because the add sample data buttons and actions were removed.

**Comment by AlexRuiz7** [comment]
Depends on #35

File: src/plugins/home/public/application/components/home.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ import {
 import { HOME_APP_BASE_PATH } from '../../../common/constants';
 import { FeatureCatalogueCategory } from '../../services';
 import { getServices } from '../opensearch_dashboards_services';
-import { AddData } from './add_data';
+// import { AddData } from './add_data';
```
Comment (Desvelao) [inline src/plugins/home/public/application/components/home.js:43]: **suggestion**: we could remove it if this is not needed.

File: src/plugins/home/public/application/components/home.js [hunk 2]
```diff
@@ -173,9 +176,12 @@ export class Home extends Component {
                 : 'homData--expanded'
             }`}
           >
+            {/*
+            Disable the card to add sample data on the home page
             <EuiFlexItem>
               <AddData addBasePath={addBasePath} features={addDataFeatures} />
             </EuiFlexItem>
```
Comment (Desvelao) [inline src/plugins/home/public/application/components/home.js:183]: **question**: Why was this disabled? Is this coming from the original source code or we did some changes previously to this pull request?

File: src/plugins/home/public/application/components/home_app.js [hunk 3]
```diff
@@ -54,41 +54,44 @@ export function HomeApp({ directories, solutions }) {
...
+  //     <Tutorial
+  //       addBasePath={addBasePath}
+  //       isCloudEnabled={isCloudEnabled}
+  //       getTutorial={getTutorial}
+  //       replaceTemplateStrings={replaceTemplateStrings}
+  //       tutorialId={props.match.params.id}
+  //       bulkCreate={savedObjectsClient.bulkCreate}
+  //     />
+  //   );
+  // };
```
Comment (Desvelao) [inline src/plugins/home/public/application/components/home_app.js:84]: **question** Do we want to keep this commented code or should we remove it instead?

File: src/plugins/opensearch_dashboards_overview/public/components/getting_started/getting_started.tsx [hunk 4]
```diff
@@ -113,6 +113,8 @@ export const GettingStarted: FC<Props> = ({ addBasePath, isDarkTheme, apps }) =>
 
             <EuiSpacer size="xl" />
 
+            {/*
+            Disable the overview button for adding sample data
             <RedirectAppLinks application={application}>
               <EuiButton
                 fill
```
Comment (Desvelao) [inline src/plugins/opensearch_dashboards_overview/public/components/getting_started/getting_started.tsx:120]: Same than https://github.com/wazuh/wazuh-dashboard/pull/23/files#r1172448788

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
Test: ✅ 

Captura de pantalla 2023-04-27 a la(s) 13 05 52
Captura de pantalla 2023-04-27 a la(s) 13 05 44
Captura de pantalla 2023-04-27 a la(s) 13 05 38

**Review by Machi3mfl — APPROVED** [review]
Test: :heavy_check_mark: 
CR: :heavy_check_mark: 

### Before

image

image

### After

image

**Pull request merged**